packages/solana/src/client.ts

```diff
@@ -281,19 +290,17 @@ export class Web3Modal extends Web3ModalScaffold {
   private async syncAccount() {
     const address = SolStoreUtil.state.address
     const chainId = SolStoreUtil.state.currentChain?.chainId
-    const isConnected = SolStoreUtil.state.isConnected
-    this.resetAccount()
 
-    if (isConnected && address && chainId) {
+    if (address && chainId) {
```

Comment:
is this ok? why remove the isConnected check?

Reply:
It is redundant with `address` and it blocking to sync account while inside the `connect` function that only sets `isConnected` when the full connection is done